crates/bench/benches/special.rs

```diff
@@ -62,20 +63,23 @@ fn custom_db_benchmarks(c: &mut Criterion) {
     let module = runtime.block_on(async { BENCHMARKS_MODULE.load_module(config, None).await });
     let mut group = c.benchmark_group("special");
     // This bench take long, so adjust for it
-    group.measurement_time(Duration::from_secs(60 * 2));
     group.sample_size(10);
 
+    let init_db: OnceLock<()> = OnceLock::new();
     for n in [10, 100] {
         let args = sats::product![n];
         group.bench_function(&format!("db_game/circles/load={n}"), |b| {
-            b.iter_custom(|iters| {
+            // Initialize outside the benchmark so the db is seed once, to avoid to enlarge the db
+            init_db.get_or_init(|| {
                 runtime.block_on(async {
                     module
-                        .call_reducer_binary("init_game_circles", args.clone())
+                        .call_reducer_binary("init_game_circles", sats::product![100])
                         .await
                         .unwrap()
                 });
+            });
 
+            b.iter_custom(|iters| {
```

Comment:
Do we still need `iter_custom` after the initializer change? It seems that we can now instead use regular `b.iter`.

Reply:
Ok, removed it, and also I made both benches to be the same logic (circles have another inner iteration).